lib/rules/no-capital-arguments.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE_CAPITAL = 'Capital argument names is not supported';
+function ERROR_MESSAGE_RESERVED(name) {
+  return `${name} is reserved argument name, try to use another`;
+}
+
+const AllowedPrefix = /[a-z]/;
+const RESERVED = new Set(['@arguments', '@args', '@block', '@else']);
+
+module.exports = class NoCapitalArguments extends Rule {
+  isReserved(name) {
+    return RESERVED.has(name);
+  }
+  visitor() {
+    return {
+      PathExpression(node) {
+        if (node.data) {
+          let part = node.parts[0] || '';
+          let firstChar = part.charAt(0);
+          let isReserved = this.isReserved(`@${part}`);
+          if (!AllowedPrefix.test(firstChar) || isReserved) {
+            if (this.mode === 'fix' && !isReserved) {
+              node.original = `@${firstChar.toLowerCase()}${part.slice(1)}${node.parts
+                .slice(1)
+                .join('.')}`;
```

Comment:
I'd rather not have a fixer at all. Changing this to a lowercase value doesn't really fix anything (unless your callsites are all passing the argument with a lowercase as well).

Reply:
fixers removed